src/gui/instrument/InstrumentFunctionViews.cpp

```diff
@@ -145,14 +145,15 @@ InstrumentFunctionArpeggioView::InstrumentFunctionArpeggioView( InstrumentFuncti
 	m_arpGateKnob->setLabel( tr( "GATE" ) );
 	m_arpGateKnob->setHintText( tr( "Arpeggio gate:" ), tr( "%" ) );
 
+	constexpr int labelFontSize = 10;
```

Comment:
Same here.

Reply:
Similar reason as above. To ensure that all labels change their sizes consistently if it needs to be changed.